src/open_inwoner/openzaak/utils.py

```diff
@@ -52,6 +53,54 @@ def is_zaak_visible(zaak: Zaak) -> bool:
     return is_object_visible(zaak, config.zaak_max_confidentiality)
 
 
+def get_role_identification_display(rol: Rol) -> str:
```

Comment:
this comment is optional, but it looks like this function will be better as `Rol` method

Reply:
If this works how we want we could move it to the zgw consumers.